guides/create-graphql-mutation.md

```diff
@@ -0,0 +1,233 @@
+## At a glance
+
+In this guide, we’ll guide you through the steps you need to create a successful mutation in GraphQL with Mailchimp Open Commerce
+## Identify which plugin owns the mutation
+
+When adding a new mutation, the first step is to decide which plugin should own it. This is usually obvious, but not always. You should think about whether any other plugins or services will need to call your mutation. If the mutation is fundamental to the system, then it may need to go in the "core" plugin, if no better alternative exists.
+
+## Understand the difference between a plugin mutation function and a GraphQL mutation resolver
+
+See [Resolver mutations and queries vs. plugin mutations and queries in the GraphQL concepts docs](.docs/graphql-concepts.md)
+
+## Define the mutation in the schema
+
+- If it doesn't already exist, create `schemas` folder in the plugin, and add an `index.js` file there.
+- If it doesn't already exist, create `schema.graphql` in `schemas` in the plugin.
+- Import the GraphQL file into `index.js` and default export it in an array:
+
+    ```js
+    import importAsString from "@reactioncommerce/api-utils/importAsString.js";
+
+    const schema = importAsString("./schema.graphql");
+
+    export default [schema];
+    ```
+
+   > NOTE: For large plugins, you can split to multiple `.graphql` files and export a multi-item array.
+
+- In the `.graphql` file, add your mutation within `extend type Mutation { }`. Add an `extend type Mutation` section near the top if the file doesn't have it yet.
+- Follow [the Relay recommendations](https://facebook.github.io/relay/graphql/mutations.htm) for mutation input arguments, which is to have only one argument named `input` that takes an input type that is the capitalized mutation name plus the suffix "Input", and to return a type that is the capitalized mutation name plus the suffix "Payload".
```

Comment:
The link does not work. Also, the text embedding the link should be 'Relay recommendations' or just "Relay"